geo_concerns/app/models/concerns/geo_concerns/image_behavior.rb

```diff
@@ -0,0 +1,31 @@
+module GeoConcerns
+  module ImageBehavior
+    extend ActiveSupport::Concern
+    include ::CurationConcerns::GenericWorkBehavior
+    include ::CurationConcerns::BasicMetadata
+    include ::GeoConcerns::BasicGeoMetadata
+
+    included do
+      aggregates :rasters, predicate: RDF::Vocab::ORE.aggregates,
+                           class_name: 'GeoConcerns::Vector',
+                           type_validator: type_validator
+      filters_association :members, as: :image_files, condition: :concerns_image_file?
```

Comment:
Yep. If you change 'GeoConcerns::Vector' to '::Vector', the test fails as it should. However if you change it to, '::Raster' it passes.